### PR TITLE
[Cognitive Services] deprecate anomalydetector

### PR DESCRIPTION
**Pull request by swathipil**
Deprecating cognitiveservices-anomalydetector package as part of https://github.com/Azure/azure-sdk-for-python/issues/34613.

Tasks done:
- [x] Update README/Changelog message to mention package is no longer being maintained, redirect to Bing Search REST API, and mention issues section of [Microsoft Bing Search SDK for Python](https://github.com/microsoft/bing-search-sdk-for-python).
- [x] remove `en-us` from all links
- [x] ~add migration guide + link to it (#35350)~ not needed for now due to low usage numbers, pr can be re-opened to merge migration guide if needed later
- [x] Run against build pipeline to make sure everything is green. -> https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3739224&view=results
- [x] Update Development Status to Inactive in setup.py
- [x] get reviews
- [x] update release date
- [x] merge pr
- [ ] update all microsoft docs referencing deprecated libs

### CODE REVIEW

File: sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.3.1 (Unreleased)
```
Comment (xiangyan99) [inline sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md:3]: Not 0.4.0?

File: sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+This package is no longer being maintained. Use the [azure-ai-textanalytics](https://pypi.org/project/azure-ai-textanalytics/) package instead for latest features and support.
```
Comment (kristapratico) [inline sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md:5]: this says textanalytics

File: sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/README.md [hunk 3]
```diff
@@ -1,23 +1,3 @@
...
-
-# Provide Feedback
-
-If you encounter any bugs or have suggestions, please file an issue in the
-[Issues](https://github.com/Azure/azure-sdk-for-python/issues)
-section of the project. 
-
-
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fazure-cognitiveservices-anomalydetector%2FREADME.png)
+This package is no longer being maintained. Use the [azure-ai-textanalytics](https://pypi.org/project/azure-ai-textanalytics/) package instead for latest features and support.
```
Comment (kristapratico) [inline sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/README.md:3]: textanalytics here too

File: sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md (hunk 2 above)
Comment (swathipil) [reply sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md:5]: 🤦 will fix, sorry!

File: sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md (hunk 1 above)
Comment (swathipil) [reply sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md:3]: Laurent just updated the guidance here: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/deprecation_process.md#124-2023-03-31

File: sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md (hunk 2 above)
Comment (swathipil) [reply sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md]: 🤦  sorry just updated!

**Review by xiangyan99 — APPROVED** [review]

**Review by kristapratico — APPROVED** [review]

**Pull request merged**